services/player-service/application/src/main/java/com/kjeldsen/player/application/usecases/GeneratePlayersUseCase.java

```diff
@@ -0,0 +1,31 @@
+package com.kjeldsen.player.application.usecases;
+
+import com.kjeldsen.player.application.repositories.PlayerPositionTendencyReadRepository;
+import com.kjeldsen.player.application.repositories.PlayerWriteRepository;
+import com.kjeldsen.player.domain.Player;
+import com.kjeldsen.player.domain.PlayerPosition;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.IntStream;
+
+@Slf4j
+@AllArgsConstructor
+@Component
+public class GeneratePlayersUseCase {
+
+    private final PlayerWriteRepository playerWriteRepository;
+    private final PlayerPositionTendencyReadRepository playerPositionTendencyReadRepository;
+
+    public void generate(int numberOfPlayers) {
+        log.info("Generating {} players", numberOfPlayers);
+        IntStream.range(0, numberOfPlayers)
+            .mapToObj(i -> playerPositionTendencyReadRepository.get(PlayerPosition.random()))
+            .map(positionTendencies -> Player.generate(positionTendencies, 200))
```

Comment:
whats 200? maybe constant? env var?

Reply:
Maybe it can be set as a setting to store in database, so admin users can change.